app/src/main/java/app/foodfinderapp/dto/Review.kt

```diff
@@ -2,7 +2,11 @@ package app.foodfinderapp.dto
 
 import android.media.Rating
 
-data class Review(var reviewID: Int, var reviewTitle: String, var reviewDescription: String, var reviewRating: Float) {
+data class Review(
```

Comment:
it is better to leave trailing commas so that when you want to update the dto and so i changed the format of the dtos to match this 
https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Trailing commas actually mean something different.  It means, leaving a comma after the last parameter (var reviewRating: Float, in this case).